src/test/scala/firrtl_interpreter/ConcreteSpec.scala

```diff
@@ -275,32 +275,111 @@ class ConcreteSpec extends FlatSpec with Matchers {
   behavior of "not"
 
   it should "flip bits of UInts" in {
+
+    def compare(s1: String, s2: String): Boolean = {
```

Comment:
I find "compare" to be misleading. Maybe rename this `bitwiseXorReduction` (pedantically) or `xorr` (tersely) to avoid confusion that this isn't checking for equality.

Reply:
Fixing this now.